internal/devtools_browser.py

```diff
@@ -766,6 +766,46 @@ def process_command(self, command):
             self.devtools.type_text(command['target'])
         elif command['command'] == 'keypress':
             self.devtools.keypress(command['target'])
+        elif command['command'] == 'mouseclick':
+            if 'target' in command:
+                target = command['target']
+                separator = target.find('=')
+                if separator == -1:
+                    separator = target.find("'")
+                if separator >= 0:
+                    attribute = target[:separator]
+                    attr_value = target[separator + 1:]
+                    mouseclickError = False
```

Comment:
this variable seems unused, can be removed?

Reply:
Removed the variable.